layouts/partials/navbar.html

```diff
@@ -29,5 +29,7 @@
 			{{ end }}
 		</ul>
 	</div>
+	{{ if .Site.Params.ui.sidebar_search_disable }}
```

Comment:
There's a problem there which I don't know if I can solve. If I don't add that condition, both search fields will be visible and the instant results will only display for the main search, not the sidebar's. I will follow you recommendation though (and only use the default search).

Reply:
Any follow up on this, @LisaFC ?

Reply:
Aaaaaah, right, I see what's going on. So as you've written it here, it means that the top search field only displays if the sidebar search is disabled (which is what you want for local search and Algolia), but they actually both work fine displayed at the same time if you're using a Google custom search (as you can see on [docsy.dev)](https://docsy.dev)) because it doesn't have an instant results feature.

At the moment for Algolia search we just tell people to disable the sidebar search as it doesn't currently work with Algolia, I think for now we can just do the same for local search. I'll have a think for a future iteration whether we can make the sidebar search automatically disappear if you're using anything except Google custom search.